scss/components/table.scss

```diff
@@ -82,20 +83,19 @@ $block: #{$fd-namespace}-table;
             }
         }
     }
-    tbody {
-        tr {
+    &__body {
+        &-row {
```

Comment:
Are these going to win specificity wars with some generic element selector like `table tbody tr` etc?

#ihatetables

Reply:
I think the goal is to not have these styles bleed over into other components, hence the making of these classes instead of using the element selectors